tsconfig.json

```
@@ -7,12 +7,12 @@
     "moduleResolution": "node",
     "noEmit": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["ES2021"],
+    "lib": ["ES2023"],
     "module": "commonjs",
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
-    "target": "ES2021"
+    "target": "ES2023"
```

I don't think `target` works with `ES2023` yet. You can try `ES2022`

```suggestion
    "target": "ES2022"
```

Or you can use `ESNext` so that bump `typescript` will also bump the target.